packages/teams-js/src/public/interfaces.ts

```diff
@@ -153,20 +154,29 @@ export interface TeamInformation {
  * Represents OS locale info used for formatting date and time data
  */
 export interface LocaleInfo {
+  /** Represents the platform on which the app is running. */
```

Comment:
> /** Represents the platform on which the app is running. */ [](http://example.com/codeflow?start=1&length=60)

How is this different than the `HostClientType`? And why are windows and macos the only choices?

Reply:
Should there be documentation as to what apps should expect if they are running on iOS, iPadOS, or Android?